account.js

```diff
@@ -112,11 +112,43 @@ class Account {
         }));
     }
 
+    /**
+     * Adds a new unrestricted key to the owners account.
+     *
+     * @param {string} ownersAccountId id of the owner's account.
+     * @param {string} publicKey public key encoded in bs58
+     */
+    async addAccountKey(ownersAccountId, publicKey) {
+        const nonce = await this.nearClient.getNonce(ownersAccountId);
+        publicKey = bs58.decode(publicKey);
+        return this.nearClient.signAndSubmitTransaction(ownersAccountId, AddKeyTransaction.create({
+            nonce,
+            originator: ownersAccountId,
+            newKey: publicKey,
+        }));
+    }
+
+    /**
+     * Removes given unrestricted key from the owners account.
```

Comment:
This will also remove an accessKey with the given public key. `removeAccessKey` does the same.

Reply:
See here https://github.com/nearprotocol/nearlib/pull/37